pkg/internal/apply/layerApplier.go

```diff
@@ -278,8 +275,8 @@ func (a KubectlLayerApplier) getSourceResources(layer layers.Layer) (hrs []*helm
 		return nil, err
 	}
 
-	//output, err := a.kubectl.Apply(sourceDir).WithLogger(layer.GetLogger()).Run()
-	output, err := a.kubectl.Apply(sourceDir).WithLogger(layer.GetLogger()).DryRun()
+	output, err := a.kubectl.Apply(sourceDir).WithLogger(layer.GetLogger()).Run()
```

Comment:
You don't want to do this.  The DryRun call reads JSON out of the source directory without actually applying it.  As of right now, we are only applying and managing HelmRelease resources so we don't actually want to apply everything in the directory, and I would prefer to add owner refs _before_ creating or updating resources in the actual cluster.